app/src/main/AndroidManifest.xml

```diff
@@ -78,7 +78,7 @@
             android:name=".contributions.MainActivity"
             android:icon="@mipmap/ic_launcher"
             android:label="@string/app_name"
-            android:configChanges="screenSize|keyboard" />
+            android:configChanges="screenSize|keyboard|orientation" />
```

Comment:
I wouldn't do [this](https://stackoverflow.com/a/7990543), it is generally just lying to yourself about things working

Reply:
With the way things are currently, if not this, a lot of implementations will need to be changed, which I think is beyond a scope of this PR, I do understand what you are saying, but I think we can create a separate issue for this, which would need a refactor of MediaDetailsFragment, which as of now is not independent.

Reply:
Code like this hides issues like this #3458 , so there are already issues to fix this.

I was only saying to get rid of the `orientation` addition, if you needed to add it to make Room work then that makes no sense to me.

Reply:
I totally understand this and of course, room does not need the orientation thing, it's that, the MediaDetailsFragment is not completely independent, it needs a provider to show media details which we loose on orientation changes, and can be solved in a separate PR, not necessarily as a part of this PR which already includes some major changes

Reply:
Also, there is a crash on master on orientation changes, which can be easily handles by this (of-course there are better ways), but this is the only one requiring the minimal amount of change. I am sure someone will pick this up, but as I said before fixing that is beyond the scope of this PR.

Reply:
There is a crash on master #3458 if it _ever_ gets recreated. It is not about orientation. This doesn't make it better and we will not be releasing without fixing this bug. This ticket isn't about applying this bandaid so I would remove it but I am happy to let it slide, and then we remove this line whenever we fix it.

Reply:
I was not referring to this crash, the existing master is crashing on changing the device's orientation, in MediaDetailsFragment, this thing was previously there, "configChanges="*|orientation" and got removed in one of the PR's (probably mine), thats why I readded this so that at least the build is stable. I agree with your logic that there should be a better way to handle this but all I am saying that a separate issue should be created and it should be taken up from there

Reply:
Yeah, I am marking this resolved, will be fixed later